assets/scripts/theme.js

```diff
@@ -146,6 +148,68 @@ class Theme {
         return document.documentElement.classList.contains(docClass);
     }
 
+    /**
+     * Finds parent element with data-cmd attribute.
+     * 
+     * @param {object} Target element.
+     */
+    findCmdAttribute(element) {
+
+        let cmdAttr, cmdCtrl, hrefAttr;
+        let foundAttr = false;
+        let foundLink = false;
+
+        while(element && element.nodeName && element.getAttribute) {
+
+            // Find data-cmds
+            if (!foundAttr) {
+                cmdAttr = element.getAttribute('data-cmd');
+                cmdCtrl = element.getAttribute('data-ctrl');
+
+                if (cmdAttr && cmdCtrl) {
+                    foundAttr = { cid:cmdAttr, el:element, ctrl:cmdCtrl };
+                }
+            }
+
+            // Find links
+            if (!foundLink) {
+                hrefAttr  = element.getAttribute('href');
+
+                if (hrefAttr) {
+                    foundLink = { href:hrefAttr, el:element };
+                }
+            }
+            element = element.parentNode;
+        }
+
+        return {
+            cmd : foundAttr,
+            link : foundLink
+        }
+    }
+
+    /**
+     * Add global listener to listen click events. If clicked dom element or parent node 
+     * has data-cmd and data-ctrl attributes, call the corresponding method 
+     * in defined controller, if exists.
+     */
+    addGlobalListener() {
+        jQuery(document).on('click', e => {
+            const captured = this.findCmdAttribute(e.target);
+
+            if (captured) {
```

Comment:
It seems this will always evaluate to `true` since `findCmdAttribute` always returns a non-empty object. Fix `findCmdAttribute` to prevent this from happening.